string time
string uuid
string consent
---
bool result
